tests/TestRunner/upgrade_test.py

```diff
@@ -20,7 +20,7 @@
 
 CLUSTER_ACTIONS = ["wiggle"]
 HEALTH_CHECK_TIMEOUT_SEC = 5
-PROGRESS_CHECK_TIMEOUT_SEC = 30
+PROGRESS_CHECK_TIMEOUT_SEC = 60
```

Comment:
If it is so slow you may also try setting numOperationsForProgressCheck to a lower value. The default is 10, we can set it to 3 or maybe even just 1

Reply:
Yeah i mean ideally I want to fix the root cause of the slowness